app/models/cuisine.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: cuisines
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  icon       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Cuisine < ApplicationRecord
+
+  validates_presence_of :name
```

Comment:
you can use a more succinct syntax: `  validates :name, presence: true, length: { maximum: 70 }`